authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/websub/impl/OndemandTemplateEventPublisher.java

```diff
@@ -135,9 +135,9 @@ private void sendEvents(BaseRequestDTO baserequestdto, String headerSignature, O
 			eventData.put(REQUESTDATETIME, DateUtils.formatToISOString(DateUtils.getUTCCurrentDateTime()));
 			eventData.put(INDIVIDUAL_ID,
 					encryptIndividualId(baserequestdto.getIndividualId(), partnerDataCert.get().getCertificateData()));
-			eventData.put(AUTH_PARTNER_ID, partner.get().getPartnerId());
+			eventData.put(AUTH_PARTNER_ID, partner.isPresent() ? partner.get().getPartnerId() : null);
```

Comment:
The same code can be replaced with ` partner.map(PartnerDTO::getPartnerId).orElse(null)`. Please check.